csv/src/test/java/com/fasterxml/jackson/dataformat/csv/deser/StreamingCSVReadTest.java

```diff
@@ -194,9 +198,9 @@ private CsvParser _parser(String csv, boolean useBytes, CsvSchema schema)
     {
         CsvParser p;
         if (useBytes) {
-            p = CSV_F.createParser(new ByteArrayInputStream(csv.getBytes("UTF-8")));
+            p = csvFactory().createParser(new ByteArrayInputStream(csv.getBytes("UTF-8")));
```

Comment:
There may be a helper method `utf8Bytes(String)` in base test class (and if not can copy from `BaseTest` of `jackson-core`).

Reply:
done